presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConnectorModule.java

```diff
@@ -88,4 +99,24 @@ protected Type _deserialize(String value, DeserializationContext context)
             return typeManager.getType(TypeId.of(value));
         }
     }
+
+    public void bindTopicSchemaProviderModule(String name, Module module)
+    {
+        install(installModuleIf(
+                KafkaConfig.class,
+                kafkaConfig -> name.equalsIgnoreCase(kafkaConfig.getTopicSchemaProvider()),
+                module));
+    }
+
+    @Provides
+    @Singleton
+    public TopicSchemaProvider getTopicSchemaProvider(KafkaConfig config, Map<String, TopicSchemaProvider> topicSchemaProviders)
```

Comment:
Why do you need that? `TopicSchemaProvider ` should be provided by `FileTopicSchemaProviderModule` and any other `*TopicSchemaProviderModule`. There should be no need for map binder. 

See `io.prestosql.plugin.hive.metastore.HiveMetastoreModule`, where each modules provides different implementaiton of `HiveMetastore`.